### ISSUE TITLE
Errors trying to use with Vite & Vue 3

### ISSUE THREAD

**Issue by wyrd-code** [Saul-Mirone/milkdown#47]
Hi,

I tried to implement Milkdown in my project as is suggested in the docs, but I am getting errors. For example, if I use `import { Editor } from '@milkdown/core`

It will throw:  `Error: ENOENT: no such file or directory 'node_modules/@milkdown/core/src/index.ts'`.

If I replace that with `import { Editor } from '@milkdown/core/lib/editor';` then it works. 

Same thing happens for other imports, like `@milkdown/preset-commonmark`.

I am using vite (2.3.3) with typescript and vue (3.0.11).

**Comment by Saul-Mirone** [comment]
Can you provide a sample project? I tried the latest version of milkdown and it works well.

**Comment by wyrd-code** [comment]
I created a small reproduction project using the latest vite vue-ts template, and trying to implement the editor component using classing sfc vue style. I didn't get to my own bug, because I ran into another one before that.

```
Uncaught ReferenceError: React is not defined
    at Proxy.<anonymous> (Editor.tsx:60)
```

If we can get past that bug, I can try and recreate the same tsconfig stuff I have in my original project after that.

Here is the link:

https://github.com/wyrd-code/milkdown-import-reproduction

**Comment by Saul-Mirone** [comment]
Fixed, you can try @milkdown/vue@4.3.3,
BTW, there is some issues in your code, the props should be dynamic
```vue
<template>
  <VueEditor :editor="editor" />
</template>

<script lang="ts">
import { defineComponent } from 'vue';
import { Editor } from '@milkdown/core';
import { VueEditor, useEditor } from '@milkdown/vue';
import { commonmark } from '@milkdown/preset-commonmark';

import '@milkdown/theme-nord/lib/theme.css';

export default defineComponent({
  name: 'Editor',
  components: {
    VueEditor,
  },
  setup: () => {
    const editor = useEditor((root) => new Editor({ root }).use(commonmark));

    return {
      editor,
    };
  }
})
</script>
```
